helper/changelog_generator.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+import os
+import subprocess
+import sys
+import re
+import pytz
+import dateutil.parser
+
+# Date since then we want to ask the git history for data
+since_date = None
+
+# List of git history log entries
+log_data = []
+
+# Author and Date
+log_author = None
+log_date = None
+
+# changelog header line
+log_start = '-' * 67 + os.linesep
+
+# timezone
+timezone = 'Europe/Berlin'
```

Comment:
Is this necessary? The timezone information has always been encoded in the commit dates, so this shouldn't be required. If you're going to force timezone conversions (which, you shouldn't!), use `UTC`.

Reply:
> Is this necessary?

I had hoped it wouldn't be. You are right the tzinfo is set in the date by git but the conversion via strftime using the %Z expansion does not work. It only worked for me on localizing and that required a timezone to be set. I'll investigate further but haven't found a solution so far

Reply:
Found a solution, will come up with new changes